plugin/al/translators/MayaReference.cpp

```diff
@@ -191,11 +204,11 @@ MStatus MayaReferenceLogic::update(const UsdPrim& prim, MObject parent) const
     }
   }
 
-  MString rigNamespaceM(rigNamespace.c_str(), rigNamespace.size());
-
   MFnDagNode parentDag(parent, &status);
   AL_MAYA_CHECK_ERROR(status, "failed to attach function set to parent transform for reference.");
 
+  MString rigNamespaceM(rigNamespace.c_str());
```

Comment:
Prefer: 
```c++
MString rigNamespaceM(rigNamespace.c_str(), rigNamespace.size());
```

You have the size of the string, so rather than invoking an additional strlen, pass the size to the ctor.

Reply:
Fixed.